fetch.bs

```diff
@@ -5938,6 +5938,25 @@ initially a new {{AbortSignal}} object.
 
 <hr>
 
+<p>To
+<dfn export lt="create a Request object|creating a Request object">create a <code>Request</code> object</dfn>,
+given a <a for=/>request</a> <var>request</var> and header guard <var>guard</var>, run these steps:
```

Comment:
```suggestion
given a <a for=/>request</a> <var>request</var> and <a for=Header>guard</a> <var>guard</var>, run these steps:
```

Just so it's easy to discover the acceptable values

Reply:
Unfortunately this is wrong as guard is a member of an object and not a standalone concept. I could make it a standalone concept so it could be used as type here though. Maybe that's worth it.

Reply:
Huh, I didn't know it was wrong to do that (given there's no `this` use in the definition).

Reply:
(having it be linkable to discover the expected values seems worth it though)